### ISSUE TITLE
Check for unused private members

### ISSUE THREAD

**Issue by socketpair** [PyCQA/pylint#3442]
```python
class Testw:
    def __init__(self):
        self._x = 1
    def __somefun(self):
        return self._x + 1
```
Should generate error "Private function defined but not used". Probably the same for module level functions.

**Comment by socketpair** [comment]
https://github.com/PyCQA/pylint/pull/4504
